_posts/2020/08/2020-08-31-governance.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Governance"
+date: 2020-08-31
+---
+
+I had another good conversation with [Mike Hoye](http://exple.tive.org/blarg/) this week
+about life, the Internet, and everything.
+I always learn a lot talking with him,
+and what I realized this time is that *governance should be discoverable*.
+Pretty much every open source project has a `LICENSE.md` file in their root directory these days,
+and most now have a `CONDUCT.md` file as well.
+These files tell passers-by how they're allowed to use the software
+and what social norms they're required to adhere to if they want to contribute to it.
+
+But how can they find out who decides what the project is going to do next,
+or who gets a say in deciding that?
+Large organizations like the [Python Software Foundation](https://www.python.org/psf/)
+[typically have bylaws](https://arxiv.org/abs/2005.10063)---as registered non-profits,
+they're legally required to---but
+[most smaller don't explain any of this](https://opensource.com/open-organization/18/4/new-governance-model-research).
+They may have a `CONTRIBUTING.md` file,
+but that usually describes how to set up a build environment and how to file bug reports,
+not how decisions are made or by whom.
+
+Spelling this out facilitates
+[legitimate peripheral participation](https://en.wikipedia.org/wiki/Legitimate_peripheral_participation),
+which [helps projects grow](https://journals.plos.org/ploscompbiol/article?id=10.1371%2Fjournal.pcbi.1007296).
+More importantly,
+it's essential to making a project genuinely inclusive.
```

Comment:
Is there a pithy description of "it" here (e.g. "a transparent decision-making process")? I'm not wed to that wording at all, but, since you started the paragraph with "this" (in _Spelling this out…_), it might be helpful here.